packages/katello/nodejs-downshift/nodejs-downshift.spec

```
@@ -4,18 +4,26 @@
 %global npm_name downshift
 
 Name: %{?scl_prefix}nodejs-downshift
-Version: 1.31.15
-Release: 2%{?dist}
+Version: 5.4.7
+Release: 1%{?dist}
 Summary: A set of primitives to build simple, flexible, WAI-ARIA compliant React autocomplete components
 License: MIT
 Group: Development/Libraries
-URL: https://github.com/paypal/downshift#readme
+URL: https://github.com/downshift-js/downshift#readme
 Source0: https://registry.npmjs.org/%{npm_name}/-/%{npm_name}-%{version}.tgz
 %if 0%{?scl:1}
 BuildRequires: %{?scl_prefix_nodejs}npm
 %else
 BuildRequires: nodejs-packaging
 %endif
+Requires: npm(@babel/runtime) >= 7.10.2
+Requires: npm(@babel/runtime) < 8.0.0
+Requires: npm(compute-scroll-into-view) >= 1.0.14
+Requires: npm(compute-scroll-into-view) < 2.0.0
+Requires: npm(prop-types) >= 15.7.2
+Requires: npm(prop-types) < 16.0.0
+Requires: npm(react-is) >= 16.13.1
+Requires: npm(react-is) < 17.0.0
```

I think this should fix it:
```suggestion
Requires: %{?scl_prefix}npm(babel-runtime) >= 7.10.2
Requires: %{?scl_prefix}npm(babel-runtime) < 8.0.0
Requires: %{?scl_prefix}npm(compute-scroll-into-view) >= 1.0.14
Requires: %{?scl_prefix}npm(compute-scroll-into-view) < 2.0.0
Requires: %{?scl_prefix}npm(prop-types) >= 15.7.2
Requires: %{?scl_prefix}npm(prop-types) < 16.0.0
Requires: %{?scl_prefix}npm(react-is) >= 16.13.1
Requires: %{?scl_prefix}npm(react-is) < 17.0.0
```

---

No it doesn't fix it, but it's a good point: those SCL prefixes are needed.